packages/grafana-ui/src/components/Select/SelectBase.tsx

```diff
@@ -293,6 +293,8 @@ export function SelectBase<T>({
             return (
               <Icon
                 name="times"
+                data-testid="select-clear-value"
```

Comment:
I think it'd be better from the accessibility point of view to have this as an `IconButton` or if that doesn't work, add `role="button"` to the icon. Also in this case `aria-label` instead of `data-testid` would be better as the button doesn't have any text inside of it so it's not picked up by screen readers.

Reply:
yea, was thinking the same. I tried IconButton first but was not ideal.

Reply:
When using Button or IconButton the preventDefault and stopPropagation does not work (stop select from opening)